src/main/java/com/segment/analytics/android/integrations/amplitude/AmplitudeIntegration.java

```diff
@@ -175,7 +184,6 @@ public void identify(IdentifyPayload identify) {
   }
 
   private void handleTraits(Traits traits) {
-    Identify identify = new Identify();
```

Comment:
This shouldn't be moved to a global variable. It's a payload - so this will leak information between calls.

Reply:
Total rookie move. Sorry.